api_docs/changelog.md

```diff
@@ -20,6 +20,11 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 7.0
 
+**Feature level 164**
+
+* [`DELETE /realm/emoji/{emoji_name}`](/api/delete-custom-emoji): New API endpoint to delete
+  custom emoji from user's organization.
```

Comment:
This is wrong -- the change is that the status code went from 400 to 404; the API endpoint' s behavior has otherwise been the same since feature level 1; it just wasn't documented before. We don't document the history of the API docs here; just the history of the API itself.

Reply:
Thanks for clearing that out!